studio/src/mas-content.js

```diff
@@ -0,0 +1,119 @@
+import { LitElement, html, nothing } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import StoreController from './reactivity/storeController.js';
+import Store from './store.js';
+import './mas-fragment.js';
+import Events from './events.js';
+
+class MasContent extends LitElement {
+    createRenderRoot() {
+        return this;
+    }
+
+    constructor() {
+        super();
+        this.goToFragment = this.goToFragment.bind(this);
+    }
+
+    loading = new StoreController(this, Store.fragments.loading);
```

Comment:
having a controller per responsive property looks overkill to me. I find it too verbose and as the codebase grows, it can become hard to wrap the head around.